### DISCUSSION TITLE
How many months of a driver's life are spent waiting for red lights to turn green? 🚦

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#802] [Galaxy Trivia] [answered]
*Based on a European study*

**Comment by fathiputera** [comment, marked as answer]
According to a study conducted by the European satellite navigation company TomTom, the average person spends 20 minutes each day waiting at red lights. The average person spends six months of their life waiting for red lights to turn green

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [🥉 Galaxy Brain](https://github.com/fathiputera?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Silver 🥈 badge next by answering 16 questions. Don't forget to post your own questions too! 😊🎉

**Comment by peterbaccaro** [comment]
The average person spends **six months of their life** waiting for red lights to turn green according to a study conducted by the European satellite navigation company TomTom.

https://maltadaily.mt/the-average-person-waits-spends-six-months-of-lifetime-waiting-for-traffic-signals-to-change/

**Comment by SANKARAMDAS** [comment]
The average person will spend six months of their life waiting for red lights to turn green.
https://www.keremeosreview.com/news/morning-start-do-you-know-how-much-of-your-life-is-spent-waiting-at-red-lights/
